src/WP_CLI/SearchReplacer.php

```diff
@@ -83,7 +83,7 @@ private function run_recursively( $data, $serialised, $recursion_level = 0, $vis
 				}
 			}
 
-			$unserialized = @unserialize( $data );
+			$unserialized = is_string( $data ) ? @unserialize( $data ) : false;
 			if ( is_string( $data ) && false !== $unserialized ) {
```

Comment:
The second `is_string()` can be omitted here, as `$unserialized` is guaranteed to be `false` if `is_string( $data )` is `false`.
```suggestion
			if ( false !== $unserialized ) {
```